database/sql/V073__track_event_indexing_progress.sql

```diff
@@ -0,0 +1,32 @@
+-- Create new table to store highest block that was processed for a given
+-- event source.
+-- This is useful for indexing very rare events. So far we would store only
+-- actually indexed events. On restarts we would look up the last stored
+-- event and continue indexing from there. If there can be a week or more
+-- between 2 events we could end up having to re-index 1 week worth of blocks
+-- on every restart until a new event gets indexed which would move the
+-- indexing "checkpoint" to the tip of the chain.
+CREATE TABLE last_processed_blocks (
+    -- string the identifies which event index the row keeps track of
+    index        TEXT PRIMARY KEY,
```

Comment:
nit: i would rather name this `event`, as for index I expected number and not text

Reply:
Then it should probably be `contract` since this applies to multiple events indexed from the same contract.